GUIDELINES.md

```diff
@@ -0,0 +1,149 @@
+  pub-year: 2019
+  category: news
+  comments: true
+  ---
+  ```
+
+  * **layout**: Defines style settings for different types of contents. All blog posts use the *posts* layout
+  * **author**: Sets the author's name, will publicly appear on the post. As a rule of thumb, use your GitHub username, Twitter handler or any other identifier known in the community
+  * **title**: The title for your blog post
+  * **description**: Short extract of the blog post
+  * **navbar_active**: Defines settings for the navigation bar, type *Blogs* is the only choice available
+  * **pub-date**: Month and day, together with *pub-year* form the date that will be shown in the blog post as the date it was published, must match the date on the file name
+  * **pub-year**: Blog post publication year, must match the year in the file name
+  * **category**: Array of categories for your blog post, some common ones are community, news and releases, as last resort, use uncategorized. If you'd like to add multiple categories, used *categories* instead of *category* and a [YAML list](https://en.wikipedia.org/wiki/YAML#Basic_components)
+  * **comments**: This enables comments your blog post. Please consider setting this to *true* and allow discussion around the topic you're writing, otherwise skip the field or set it to false
+
+* Blog post contents recommendation:
+
+  * Title is a very important piece of your blog post, a catchy title will likely have more readers, write a bad title and no matter how good the contents is, you'll likely get less readers
+  * After the title, write a brief introduction of what you're going to be writing about, which will help the reader to get a grasp on the topic
+  * Closing section, the same way we can add a brief introduction of what the blog post is about, it's very important to add a closing section with thoughts, upcoming work on the topic discussed, encourage readers to test something and share their findings, joining the community, ...
+
+### Pages
+
+*[Pages](https://jekyllrb.com/docs/pages/)* are located at the [pages](/pages/) directory, to create one follow these steps:
+
+* Create the markdown file, *filename.md*, in [pages](/pages/) directory
+* *Pages* also use [Front Matter](https://jekyllrb.com/docs/front-matter/), here's an example:
+
+  ```yaml
+  ---
+  layout: default
+  title: Introduction
+  permalink: /docs/
+  navbar_active: Docs
+  ---
+  ```
+
+* The fields have the same function as for blog posts, but some values are different, as we're producing different contents.
+  * **permalink** tells *Jekyll* what the output path for your page will be, it's useful for linking and web indexers
+  * **navbar_active** will add your page to the navigation bar you specify as value, commonly used values are *Docs* or *Videos*
+  * **layout**, just use *default* as value, it'll include all the necessary parts when your page is generated
+
+* As for the contents, follow the general guidelines above
+
+### Labs
```

Comment:
Have we confirmed this site will contain labs?

Reply:
From the spreadsheet we've of possible content, labs are listed ```Blog posts to troubleshoot rhhi.next (Issue vs resolution labs?)```